src/utils/media-utils.js

```diff
@@ -544,3 +544,17 @@ export function closeExistingMediaMirror() {
     });
   }
 }
+
+export function hasAudioTracks(el) {
+  if (!el) return false;
+  // At some point browsers will implement the audioTracks property
+  if (el.audioTracks !== undefined) {
```

Comment:
Minor nit, but I would have probably written this if/else chain as just a boolean expression